pyproject.toml

```diff
@@ -292,6 +292,9 @@ filterwarnings = [
     "ignore:pkg_resources is deprecated as an API.:DeprecationWarning",
     "ignore:Subclassing validator classes is not intended to be part of their public API.:DeprecationWarning",
     "ignore:Subclassing validator classes:DeprecationWarning:tableschema",
+    "ignore:datetime.datetime.utcnow() is deprecated:DeprecationWarning:dbt_common",
+    "ignore:datetime.datetime.utcnow() is deprecated:DeprecationWarning:dbt",
+    "ignore:Python 3.14 will, by default, filter extracted tar archives:DeprecationWarning:dbt_common",
```

Comment:
The warnings in our test logging output are getting out of control and making it hard to see wtf is actually going on.  We need to address various deprecation warnings #3418 and also silence the ones that aren't our problem.